int8 flag
---
float32[] joint_config